bin/check-asg-instances-inservice.rb

```diff
@@ -0,0 +1,109 @@
+#! /usr/bin/env ruby
+#
+# check-autoscaling-instances-inservice
+#
+# DESCRIPTION:
+#   Check AutoScaling Instances inService.
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: aws-sdk
+#   gem: sensu-plugin
+#
+# USAGE:
+#   ./check-autoscaling-instances-inservices.rb -r ${your_region}
+#   one autoScalingGroup
+#   ./check-autoscaling-instances-inservices.rb -r ${your_region} -g 'autoScalingGroupName'
+#
+# NOTES:
+#   Based heavily on Yohei Kawahara's check-ec2-network
+#
+# LICENSE:
+#   Peter Hoppe <peter.hoppe.extern@bertelsmann.de>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+class CheckAsgInstancesInService < Sensu::Plugin::Check::CLI
+  option :aws_region,
```

Comment:
Include Common similar to https://github.com/sensu-plugins/sensu-plugins-aws/blob/master/bin/check-eip-allocation.rb#L34